inc/project.class.php

```diff
@@ -125,23 +125,25 @@ function getRights($interface='central') {
 
    function getTabNameForItem(CommonGLPI $item, $withtemplate=0) {
 
-      if (static::canView()) {
-         $nb = 0;
-         switch ($item->getType()) {
-            case __CLASS__ :
-               $ong    = array();
-               if ($_SESSION['glpishow_count_on_tabs']) {
-                  $nb = countElementsInTable(
-                     $this->getTable(),
-                     [
-                        $this->getForeignKeyField()   => $item->getID(),
-                        'is_deleted'                  => 0
-                     ]
-                  );
-               }
-               $ong[1] = self::createTabEntry($this->getTypeName(Session::getPluralNumber()), $nb);
-               $ong[2] = __('GANTT');
-               return $ong;
+      if (!$withtemplate) {
```

Comment:
to avoid an additional indentation and as you don't use else statement, please don't write nested ``if``
An ``&&`` should be sufficient.